cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/NewCommand.java

```diff
@@ -71,9 +71,9 @@ public void execute() {
         // Check if the project name is given
         if (null == argList) {
             CommandUtil.printError(errStream,
-                    "The following required arguments were not provided:\n" +
+                    "following required arguments were not provided:\n" +
```

Comment:
I think we need the "the" here. @pubudu91 wdyt?

Reply:
Added "the".